src/helix-ui/styles/core/hx-search-assistance.less

```diff
@@ -0,0 +1,55 @@
+hx-search-assistance {
+  background-color: @gray-0;
+  border: 1px solid @gray-300;
+  box-shadow: 0px 3px 3px 0 @shadow-color;
+  display: none;
+  margin-top: 0.25rem;
+  max-height: 20rem;
+  overflow-x: hidden;
+  overflow-y: auto;
+  padding: 0.75rem;
+  position: absolute;
+  top: 0.25rem;
+  width: 34.25rem;
```

Comment:
Note: Leave as width for now instead of max-width so we can get more design info.